### ISSUE TITLE
Product List

### ISSUE THREAD

**Issue by lkoenigsberg** [department-of-veterans-affairs/va.gov-team#45134]
**What**
Develop,  and manage, a single source of truth for all OCTO products and ensure all products have an associated product outline.

**Why**
Say some good stuff here...

**Things to do**
- Finalize and communicate [definitions](https://dvagov-my.sharepoint.com/:w:/r/personal/lisa_koenigsberg_va_gov/_layouts/15/Doc.aspx?sourcedoc=%7B675F04BA-5764-4EF7-9D33-AD84907735E5%7D&file=Product%20Operations%20Definitions%20-%20DRAFT.docx&action=default&mobileredirect=true&cid=34c109b2-d031-47c8-8e77-1d23822e68cb) 
- Meet with the Platform Crew to discuss the work they've been doing and being tracked [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/44520) and [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39190)
- Update the list so that it maps to the new definitions by working with Product Owners, Crew Chiefs, and Portfolio Leads 
- Develop a method for keeping this updated and maintained
- Others??

**Comment by lkoenigsberg** [comment]
[Platform Product List landing page](https://vfs.atlassian.net/wiki/spaces/TTT/pages/2193359369/VA.gov+Product+Directory+Documentation)

**Comment by dmkosko** [comment]
8/24 - Benefits Tools, Sitewide, Account Experience, and Mobile product discussions set. Still arranging Platform and Health meetings

**Comment by dmkosko** [comment]
8/31 - Met with Matt Self and Dave Conlon this week so far, updating notes on the product list. Meeting with Account Experience this week

**Comment by dmkosko** [comment]
9/7 - Hold off on scheduling a Platform product discussion until Lisa speaks with Chris

**Comment by dmkosko** [comment]
9/15 - Draft of the comms plan shared with the group

**Comment by dmkosko** [comment]
Arranging a meeting with Platform to discuss the VFS Directory and our product list. The VFS Directory has some components that we would like to include with our list so it makes sense to meet and discuss next steps.

**Comment by lkoenigsberg** [comment]
Noticed there are no Benefits Delivery Portfolio products on the list - Virtual Regional Office (VRO) as an example.  Add this to the task list.

**Comment by dmkosko** [comment]
Added products from the Product Directory to our spreadsheet to compare the two lists

Added in forms that are associated with products (need a second set of eyes to review)

**Comment by dmkosko** [comment]
Updated Product List shared. There are a few blanks/question marks in the "Crew" column as I could not find info from GitHub or the Product Directory.

**Comment by dmkosko** [comment]
- [ ] Lisa and Dan to review Health products on our spreadsheet and reassess/confirm if they are correctly labeled as a product, feature, initiative as well as if there is a Product Lead assigned 
- [ ]  Joe Tice and Peter Hill helped create the VFS Directory site – they would have knowledge on adding/updating products.  Ray and Jason to coordinate a technical evaluation of what the process is for adding non VA.gov products to the list would look like
- [ ] Mickin to assist in determining leads for DE products. If there are questions, we will need to loop in Crew Chiefs. After that exercise, we will need to look into “deduping” our product spreadsheet and then submit to Platform if the VFS Directory is going to be the one source of truth 
- [ ]  Dan to collect the outstanding items by crew and clean up any notes if the question/task was completed
- [ ] Ray to finish the Platform items this week and even added some features to our spreadsheet.  Making sure there is a lead named for each thing.

**Comment by dmkosko** [comment]
Dan reaching out to Product Leads to add Product Outline links to our spreadsheet.

**Comment by dmkosko** [comment]
Ticket submitted to have our product definitions added to the VFS Directory for awareness (Our GitHub link will be added).

**Comment by dmkosko** [comment]
Lauren added in Product Leads to any Health items and the leads will update their items before December 2nd. 
Zach's items are updated. 
Reached out to Chante and Matt regarding their items (waiting to hear back)
Reaching out to Dave today

**Comment by dmkosko** [comment]
Platform items updated
Remaining Crews are Account Experience, Sitewide and Benefit Tools. Matt and I have exchanged messages back and forth, just waiting on his last set of answers. For Sitewide, Dave had questions about why/how some items were listed as products when nobody from Platform ever reached out to him for questions. I worked with Joe Tice to get Dave more info and I'm just waiting on him to see how he feels about some of the items in our list (He is out sick today, but plan on reaching out tomorrow if he is back online). 

@lkoenigsberg I need some assistance with Account Experience, I reached out to Chante a few times and didn't get a response. There's only three products I can't find any outline information on (Medallia, MyVA 411 Voicebot, and Performance Dashboard). 

There are a couple outstanding items noted above, but we are very close to getting this wrapped up.

**Comment by dmkosko** [comment]
Account Experience: draft product outline for MyVA 411 Voicebot will be available early next week (otherwise Account Experience is complete)
Benefit Tools: finalized remaining items with Matt, now complete
Sitewide: confirming with Dave if Benefit Hub and Content Hub should be considered products

**Comment by dmkosko** [comment]
- [ ] Dan to add a column in our spreadsheet to sort VA.gov vs. Non-VA.gov products by December 19th 
- [ ] Dan to work with Jason on seeing what we can do to update the minimum required information on the VFS Directory to include a link to the product outline, crew responsible for the work, and the Product Lead by January 6th, 2023
- [ ] Dan and Jason will determine how the fields are structured and see if there’s a way to have them reordered by January 6th 
- [ ] Lisa would like to know if there’s the potential for the VFS Product Directory to be filtered by Crew, Owner, etc? Dan to work with Jason to determine by January 6th, 2023 
- [ ] Lisa to write up who the audience is for the VFS Product Directory and how this list is used by the teams by January 6th, 2023
- [ ] Jason to provide details decision-making process for a product vs. Not during the collab cycle by January 6th, 2023   
- [ ] Dan to propose next steps to Jason on best way to go about the cleanup (I.e. should there be some type of “archive” folder to track any items we move from products to not, and would it be beneficial for a comm to be present in office hours?) by January 9th, 2023
- [ ] Dan to identify what items will be coming off the VFS Product Directory which will assist in creating the communication by January 9th, 2023 
- [ ] Dan and Jason to find the gaps and clean up the current VFS Product Directory to ensure it matches with our excel spreadsheet by January 9th, 2023

**Comment by dmkosko** [comment]
One of the items discussed is to provide Platform a communication about updating the VFS Product Directory to ensure it matches our excel list. A link to the comm is [here](https://dvagov-my.sharepoint.com/:w:/r/personal/lisa_koenigsberg_va_gov/Documents/ProductOperations/Communication%20about%20VFS%20Directory%20Updates.docx?d=w48bb3336c44949a081de5ba5607ef0cb&csf=1&web=1&e=QrtyCQurl) and will be reviewed next week

**Comment by dmkosko** [comment]
Decision making process for product vs. not below

https://dsva.slack.com/archives/C03H546LHDY/p1672870149669939?thread_ts=1672867800.880709&cid=C03H546LHDY

**Comment by dmkosko** [comment]
Regarding the in-progress action items:

Justification for reordering fields and creating a new view on the VFS Directory: link to response [here](https://dsva.slack.com/archives/C03H546LHDY/p1672963760633949?thread_ts=1672764247.667339&cid=C03H546LHDYl)
Communication to Platform Collab Cycle: Waiting on feedback from Jason and will share with Mickin once he's back from being OOO. I spoke to Shira and it may be beneficial if an additional educational form was created to provide more insight into the collab cycle for any future product requestors. Dan will add this as a separate item to our board. 
Next week we will tackle the process to remove items from the VFS Directory that are no longer needed

**Comment by dmkosko** [comment]
Sent Mickin the communication for the Platform collab cycle - waiting on feedback
Connected with Shira to add the product definitions to additional pages in the collab cycle
In the progress of scheduling a meeting with this group and Joe Tice to discuss the changes to the VFS Product Directory

**Comment by dmkosko** [comment]
EOW updates:

Sent Mickin the communication for the Platform team - waiting on feedback
Justification about VFS Directory changes – needs to be reviewed by Mickin
Reordering/Sorting/Removal – Holding off on applying any changes until we meet with Joe T on Jan 18th

**Comment by dmkosko** [comment]
Meeting with Jason Woodward today to discuss who the Platform comm should be sent to. 
Link to the VFS Directory reordering/sorting/removal initiatives is [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52276#issuecomment-1402394954)

**Comment by dmkosko** [comment]
Communication to Platform team individuals that work with product sent on 1/26. Mentioned to the group that the purpose is for awareness of the changes we're proposing. The only action is to disseminate to their team members who interact with any areas of product.

**Comment by dmkosko** [comment]
- [ ] Dan to go through the product list and determine the last times the outlines have been updated. If they have not been updated recently (~12 months), reach out to the Product Lead and discuss when it can be. Our criteria was reasons to be updated are: does the product outline reflect the current state of the product, has any new research been conducted since the last time it was updated, have any releases occurred, are there any new/updates to metrics/analytics/KPIs, and is there anything that happened recently that warrants an update (change of product mgmt team, foresee/unforeseen risks, launch strategy, or problem statement.)

**Comment by dmkosko** [comment]
For the top task, I am going to hold off on reaching out until Lisa presents the information on the product list / outline metrics on the monthly CoP call. I am working on the criteria by which we're asking Product Leads to update their outlines so that way there is a standardized discussion on why/when an outline should be updated.

**Comment by dmkosko** [comment]
Metrics presented to Charles and others on products

- There are 106 products on our spreadsheet, 95 associated with [VA.gov](http://va.gov/), 11 [non-VA.gov](http://non-va.gov/)
- 92% (98/106) of products DO have some type of outline, could be in GitHub or Confluence (all 11 [non-VA.gov](http://non-va.gov/) products do have an outline, thought it was good to mention that)
- Only 50 of the 98 product outlines (51%) have been updated since February 1, 2022

**Comment by dmkosko** [comment]
Product Outline updates can be found [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53693)

**Comment by dmkosko** [comment]
Met with Joe Tice earlier today to discuss the process to have the VFS Product Directory match our excel product list. He will need to do some digging and research to determine the best course of action for removals/additions (and loop in Product Leads and Shira G.). Joe will create tickets and send them over. The biggest question is, what to do with the products that will be removed and where should they live for historical purposes?
